client/src/pages/TestChefProfile.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useContext, useEffect } from "react";
+import defaultUserImage from "../assets/defaultUserImage.png";
+import DialogControl from "../components/Dialogs/DialogControl";
+import PlaylistAddIcon from "@material-ui/icons/PlaylistAdd";
+import {
+  RecipeContext,
+  RecipeDispatchContext,
+} from "../context/recipe-context";
+import { UserContext } from "../context/UserContext";
+import Recipe from "../components/Recipe";
+import axios from "axios";
+import { Button } from "@material-ui/core";
+import { getRecipesByChef } from "../actions/recipeActions";
+import NavBar from "../components/NavBar";
+
+const ChefProfile = () => {
+  const [open, setOpen] = useState(false);
+  const [control, setControl] = useState(null);
+
+  const { recipes } = useContext(RecipeContext);
+  const dispatch = useContext(RecipeDispatchContext);
+
+  const { userInfo } = useContext(UserContext);
+  const [profile, setProfile] = useState(null);
+
+  useEffect(() => {
+    const getProfileAndRecipes = async () => {
+      const res = await axios.get(`/api/chefProfiles/${userInfo._id}`);
+      if (res.data) {
+        setProfile(res.data.chefProfile);
+        getRecipesByChef(dispatch, res.data.chefProfile._id);
+      }
+    };
+    getProfileAndRecipes();
+  }, [dispatch, userInfo._id]);
+
+  const handleClickOpen = (e) => {
+    setOpen(true);
+    setControl(e.target.name);
+  };
+
+  const handleClose = (value) => {
+    setOpen(false);
+  };
+  const handleNewChef = async () => {
+    //make user a chef
+    await axios.put("/api/users/isChef");
+    setControl("CreateProfile");
+    setOpen(true);
+  };
+  return !profile ? (
+    <>
+      <NavBar />
+      <div>
+        <h1>Please set up a profile</h1>
+        <Button
+          type="submit"
+          variant="contained"
+          color="secondary"
+          onClick={handleNewChef}
+          style={{
+            borderRadius: "0",
+            width: "200px",
+            height: "40px",
+            marginTop: "30px",
+            textTransform: "capitalize",
```

Comment:
Try to avoid inline styling

Reply:
Hey Nellie, Thanks for your time. I am working on the comments. 
Just wanted to say, this is a test component. Because at the time I was working on recipe functionality, there was no chefProfile, I had to have a basic one to see my changes and my functionality in it-There still is no chef profile in dev-. So basically only the logic part of the component will remain(The logic for context and actions and so on. So if that's okay, can i ignore the parts that is not going to be in the final application. I will completely delete these test components when Val made his merge and I added the functionalities I want to his component.